index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Tech Academy</title>
+  <link rel="stylesheet" href="https://use.typekit.net/kjf8squ.css">
+  <script src="https://kit.fontawesome.com/67c66657c7.js"></script>
+  
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header class="primary-header">
+    <div class="container">
+          <nav class="navigation">
+            <div class="menu-button"><img id="humberger" src="img/menu.svg" alt="menu-button"></div>
+            <a href="index.html"><img class="logo"src="img/logo-2.png" alt="Manage" /></a>
+          <div class="nav-bar">
```

Comment:
![white-spacing-homepage-capstone (copy)](https://user-images.githubusercontent.com/93039359/196929119-3c9ba15a-8f99-422b-8b63-d03e3841859b.png)

- [ ] Your homepage has a horizontal scroll on 375px screen size which is Iphone screen size and is a standard for designing the mobile applications. Kindly remove that horizontal scroll and the white spacing shown in the image above :+1: